GDJS/Runtime/bitmapfontmanager.js

```diff
@@ -0,0 +1,263 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2021-present Aurélien Vivet (bouh.vivez@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+
+/**
+ * BitmapFontManager loads fnt/xml files (using `fetch`), using the "bitmapFont" resource
+ * registered in the game resources, and install the "bitmapFont" with PixiJS for using it with PIXI.BitmapText
+ * @class BitmapFontManager
+ * @memberof gdjs
+ * @param {ResourceData[]} resources The resources data of the game.
+ */
+gdjs.BitmapFontManager = function (resources) {
+  this._resources = resources;
+  /** @type Object.<string, Object> */
+  this._fontUsed = {};
+  /** @type Array.<Object> */
+  this._fontTobeUnloaded = [];
+  this._loadedFontsData = {};
+  this._defaultSlugFontName = ''; // Define once from styles in getDefaultBitmapFont()
+};
+
+gdjs.BitmapFontManager.prototype.getDefaultSlugFontName = function () {
+  return this._defaultSlugFontName;
+};
+
+// Use element from PIXI.BitmapFont for generate a unique name and replace the original by the new one in PIXI.BitmapFont
+gdjs.BitmapFontManager.prototype.generateSlugName = function (bitmapFont) {
+  return bitmapFont.font + '-' + bitmapFont.size + '-' + bitmapFont.lineHeight;
+};
+
+// We patch the installed font to use a more complex name that includes the size of the font and line height,
+// to avoid conflicts between different font files using the same font name.
+gdjs.BitmapFontManager.prototype.patchBitmapFont = function (bitmapFont) {
```

Comment:
uber nitpicking: let's call this `patchInstalledBitmapFont` to make it clear we're modifying something that is inside PIXI.BitmapFont.available, and so is "installed" (following the terms used in PixiJS).